packages/enterprise/src/license.ts

```diff
@@ -51,6 +51,7 @@ export type CommercialFeature =
   | "multi-metric-queries"
   | "no-access-role"
   | "teams"
+  | "custom-roles"
```

Comment:
This already exists further down the list.  I'm surprised Typescript didn't complain about this.  Same with the change below in this file.

Reply:
Oh yea - very weird.